src/settings.js

```diff
@@ -290,11 +290,8 @@ var IGNORE_CLOSURE_COMPILER_ERRORS = 0;
 // [link]
 var DECLARE_ASM_MODULE_EXPORTS = 1;
 
-// A limit on inlining. If 0, we will inline normally in LLVM and closure. If
-// greater than 0, we will *not* inline in LLVM, and we will prevent inlining of
-// functions of this size or larger in closure. 50 is a reasonable setting if
-// you do not want inlining
-// [compile+link]
+// If 0, prevents inlining if set to 1. If 0, we will inline normally in LLVM.
+// [compile]
```

Comment:
Is this correct? It doesn't seem to affect linking for now. It does not affect closure anymore, right?

Reply:
Yes, this just affects a clang compile flag.

And yes, good point that this does not affect Binaryen, which could be confusing. It's would be good to mention that here as well.

Also perhaps it would be good to say this is deprecated, and the user can just pass the clang flag directly?

Reply:
But even if they pass the clang flag inlining in Binaryen can still occur, which is equally confusing, no?

Reply:
I'm not necessarily saying we should make this option (or the clang flag) affect Binaryen too, but rather saying it looks commenting about `-finline-functions` here doesn't change things much because that also does not affect Binaryen.

Reply:
Yes, it is confusing, I agree.

The added warning here that it doesn't affect binaryen is useful I think. :+1: